src/main.rs

```diff
@@ -49,20 +59,11 @@ impl EventHandler for MyGame {
     }
 
     fn draw(&mut self, context: &mut Context) -> GameResult<()> {
-        let grey: Color = [0.5, 0.5, 0.5, 1.0].into();
-        let turquoise: Color = [0.00, 0.42, 0.5, 1.0].into();
+        graphics::clear(context, GREY);
 
-        graphics::clear(context, grey);
-        // Draw code here...
-        let circle = graphics::Mesh::new_circle(
-            context,
-            graphics::DrawMode::fill(),
-            mint::Point2{x: 400.0, y: 500.0},
-            200.0,
-            0.1,
-            turquoise,
-        )?;
-        graphics::draw(context, &circle, graphics::DrawParam::default())?;
+        for i in 0..self.players.len() {
```

Comment:
I think this is more idiomatic
```rust
for player in self.players.iter() {
        player.draw(context)?;
}
```

Reply:
Yeah this will work for the `draw()`.  I originally had this for both loops, but it doesn't work for 
```rust
// Do not let the players move out of the window.
for mut player in self.players.iter() {
      player.arena_bounds(arena_width, arena_height);
}
```
because player is a reference and cannot be borrowed as mutable. Maybe I can think of a better way tp do this.

Reply:
Oh this is a good meme, my design decision is based on code that you don't know exists yet.

Reply:
Nvm @vtereso informed me that `iter_mut()` exists. 🤷